alibi_detect/cd/tests/test_spot_the_diff.py

```diff
@@ -57,7 +57,7 @@ def test_stddrift(stddrift_params):
         kernel = MyKernelTF((n_features,))
     else:
         kernel = None
-    x_ref = np.random.randn(*(n, n_features)).astype(np.float32)
+    x_ref = np.random.randn(*(n, n_features))
```

Comment:
?

Reply:
I added `astype(np.float32)` because this was necessary when I added a dummy model call in `SpotTheDiffDrift` to build the model before passing to `ClassifierDrift` (prior to [d441a29](https://github.com/SeldonIO/alibi-detect/pull/739/commits/d441a293336dc6ec99d6e3888827a3332fb6939a)). We're no longer doing any dummy calls because we've reverted to cloning the model instead of getting/setting weights.

Technically the data should be float32 if we're actually going to call the model or detector, but we don't do that in this test. Just trying to keep unrelated changes to a minimum in the PR...